Sources/Plasma/FeatureLib/pfMetalPipeline/plMetalDevice.cpp

```diff
@@ -1241,12 +1240,9 @@ MTL::SamplerState* plMetalDevice::SampleStateForClampFlags(hsGMatState::hsGMatCl
 void plMetalDevice::CreateGammaAdjustState()
 {
     MTL::RenderPipelineDescriptor* gammaDescriptor = MTL::RenderPipelineDescriptor::alloc()->init();
-    MTL::Library*                  library = fMetalDevice->newDefaultLibrary();
-
-    gammaDescriptor->setVertexFunction(library->newFunction(MTLSTR("gammaCorrectVertex"))->autorelease());
-    gammaDescriptor->setFragmentFunction(library->newFunction(MTLSTR("gammaCorrectFragment"))->autorelease());
 
-    library->release();
+    gammaDescriptor->setVertexFunction(fShaderLibrary->newFunction(MTLSTR("gammaCorrectVertex"))->autorelease());
+    gammaDescriptor->setFragmentFunction(fShaderLibrary->newFunction(MTLSTR("gammaCorrectFragment"))->autorelease());
```

Comment:
Need the conditional here to bind to `gammaCorrectFragmentInPlace`?

Reply:
Unfortunately not that simple. I do have a branch though that makes the necessary changes.

https://github.com/colincornaby/Plasma/commit/4d9d984ea4c64505365dfa5724c6db539b28dbb9

Reply:
Just checking in on how you're feeling about this @dpogue - I can bring that other branch in but it might be nice to avoid the churn and leave that to a secondary PR.

Reply:
Yeah, that sounds fine to me.